docs/src/modules/ROOT/pages/enterprise-edition/enterprise-edition.adoc

```diff
@@ -0,0 +1,707 @@
+= Enterprise Edition
+:doctype: book
+:sectnums:
+:icons: font
+
+Timefold Solver Enterprise Edition is a commercial product that offers additional features,
+such as <<nearbySelection,nearby selection>> and <<multithreadedSolving,multi-threaded solving>>.
+These features are essential to scale out to very large datasets.
+
+Unlike Timefold Solver Community Edition,
+the Enterprise Edition is not open-source.
+To use Timefold Solver Enterprise Edition in production,
+you need to https://timefold.ai/company/contact/[purchase a license].
```

Comment:
Should we explicitly say that Enterprise Edition can still be used for an evaluation/development purposes?